mucommander-core/src/main/java/com/mucommander/ui/theme/ThemeData.java

```diff
@@ -1222,7 +1230,14 @@ private static Color getDefaultColor(int id, ThemeData data) {
         // Makes sure id is a legal color identifier.
         checkColorIdentifier(id);
 
-        return COLORS.get(Integer.valueOf(id)).getColor(data);
+        DefaultColor defaultColor = COLORS.get(Integer.valueOf(id));
+        if (defaultColor == null) {
+            // see the first comment for this class
+            LOGGER.debug("Default color for id {} not found, using generic default");
```

Comment:
```suggestion
            LOGGER.debug("Default color for id {} not found, using a generic default");
```
and you didn't provide the `id` here